lib/index.ts

```diff
@@ -138,7 +143,8 @@ async function startRepl(files: string[]) {
     driver,
     rerun: rerun.bind(null, files),
   });
-  replObj.on('exit', cleanup);
+  // This cleanup is called outside a mocha hook, so `timeout(ms)` does nothing.
+  replObj.on('exit', () => cleanup({timeout: () => undefined}));
```

Comment:
Is there a benefit to setting this noop here vs as a default parameter value?

Reply:
This way it's harder to forget to specify the context.

Reply:
OK!